### PR TITLE
chore: Update version, release notes for 2.20.0

### PR DESCRIPTION
**Pull request by riddell-stan**

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -6,8 +6,11 @@
  What's New
 ============
 
-TBD (TBD)
-=========
+v2.20.0.0 (15. Aug 2019)
+========================
+- Update Stan source to v2.20.0 (`release notes <https://github.com/stan-dev/stan/releases/tag/v2.20.0>`_)
+- Fix: Use prefix 'pystan_' with tempfolder (@ahartikainen)
+- Reminder: Python 2 is not supported after 2019-12-31. PyStan will phase out testing of Python 2 after this point.
```
Comment (ahartikainen) [inline doc/whats_new.rst]: - Convenience functions for external C++ (advanced users)
- possibility to set inv_metric manually (+ convenience functions to get stepsize, inv_metric and last_location/last_warmup)

Comment (ahartikainen) [reply doc/whats_new.rst]: - Experimental function to load old pickled fits, without the proper model.

Comment (riddell-stan) [reply doc/whats_new.rst]: You're right. I missed those. I'll add them.

**Comment by riddell-stan** [comment]
Should we wait until the STAN_THREADS issue is resolved before releasing? See https://github.com/stan-dev/math/issues/1306 for the issue tracking the problem.

**Comment by ahartikainen** [comment]
For pystan 2.x, no. Let's do new release if it is fixed (?)

**Comment by ahartikainen** [comment]
Closing for now.

**Pull request closed without merging**